pkgs/by-name/tr/treefmt/package.nix

```
@@ -27,13 +29,46 @@ buildGoModule rec {
     "-X github.com/numtide/treefmt/v2/build.Version=v${version}"
   ];
 
+  passthru = {
+    /**
+      Configure treefmt using structured settings.
+
+      # Type
+
+      ```
+      AttrSet -> Derivation
+      ```
+
+      # Inputs
+
+      - `name`: `String` (default `"treefmt-configured"`)
+      - `settings`: `Module` (default `{ }`)
+      - `runtimeInputs`: `[Derivation]` (default `[ ]`)
+    */
+    wrapWithConfig = callPackage ./wrap-config.nix { };
```

```suggestion
    withConfig = callPackage ./wrap-config.nix { };
```
`treefmt.withConfig` flows a bit better and is consisdtent with other packages such as `terraform.withPlugins`. But it's a matter of preference.

---

Thanks, much better.